conu/backend/docker/container.py

```diff
@@ -370,3 +370,14 @@ def exit_code(self):
         :return: int
         """
         return self.get_metadata()["State"]["ExitCode"]
+
+    def active_write(self, message):
```

Comment:
Can we rename it to `write_to_stdin`? For me, the name is really confusing and I wouldn't know what I should expect.